solanches/controller.py

```diff
@@ -37,11 +37,13 @@ def get_comercio_by_name(comercio_nome):
     assert comercio, f'Erro: comercio com nome {comercio_nome} nao cadastrado!'
     return comercio
 
+
 def remove_comercio(comercio_nome):
     assert comercio_nome and type(comercio_nome) is str, f'Erro: nome de comercio invalido'
-    comercio = Comercio.remove_comercio(comercio_nome)
-    assert comercio is not 0, f'Erro: comercio com nome {comercio_nome} nao cadastrado!'
-    return comercio
+    comercio = Comercio.get_by_name(comercio_nome) 
+    assert comercio, f'Erro: comercio com nome {comercio_nome} não removido!'
```

Comment:
```suggestion
assert comercio, f'Erro: comercio com nome {comercio_nome} não cadastrado!'
```
assim indica que o comércio que tentaram remover nem estava cadastrado.